### ISSUE TITLE
local notification Upgrade!!

### ISSUE THREAD

**Issue by eros2187** [katzer/cordova-plugin-local-notifications#754]
I would not use the server gcm or ios but it would be useful to use local notification fact I created a function that queries the server and it returns an array json but the function that run every x minutes until backgraund or app and open it all works but with the app I closed not get anything because the function closes.
In fact, I wanted to know that you are an expert if possible era somehow a function even if app and stta closed!
It would be ideal so you can use the local notification without using any server gcm and especially compatible for all platforms in my opinion
It would not be a good idea to you?
Feasible is ??

**Comment by MichelReij** [comment]
Hi Eros, I'm afraid I understand little from what you are trying to accomplish. Could you rephrase the above in simpler English?

**Comment by eros2187** [comment]
I can not perform a function of a distant type:

setInterval(function(){

 Update_push ()

}, 3000);

function Update_push (){
 //function which updates the message title push, ecc message, title, ecc.. to my server
data.title; data.text;ecc..ecc
}

cordova.plugins.notification.local.schedule({
    id: 1,
    title: "'data.title'",
    text: "'data.text'",
    firstAt: ecc..
    every: ecc..

everything works if app is bacgroud I open but if the app is closed setInterval not part !!

you can launch function 
 Update_push ()

even if the app is closed ?
how does wotzapp ??

in this way we can use push notifications as if they were no more local bypassing the usage of the server  gcm etc. would be great if you could implement in some way !!

you can do it ??
thanks for your time greetings eros

**Comment by MichelReij** [comment]
So you mean something like push notifications in reverse? A local notification fires on your phone and then this gets pushed to the server?

**Comment by eros2187** [comment]
yes I meant to have this very own local notifications as if they were no longer local but dynamically updated chart using a server !!
the problem is that the function that connects to the server only works if appa and open or in the background but does not work if app is closed

**Comment by MichelReij** [comment]
Yes, that is correct. The user needs to click the notification in order for your app to do something meaningful like updating a server.

**Comment by eros2187** [comment]
I was wondering if it was possible instead of clicking DO rate automatically expire at a timeout

**Comment by katzer** [comment]
@eros2187 I understand your use case but thats unrelated to __local__-notifications.
